### ISSUE TITLE
how to debug?

### ISSUE THREAD

**Issue by nixmeer** [lukasroegner/homebridge-nuki#57]
Hello :) 

I got a lock and an opener. Both main functionality is working but the optional sensors/bells are not. 

I do not receive a notification, if the bell rings even though RTO works so the bell is detected by the opener. Should there an accessory type like door bell? I tried activating notifications for the opener lock tile but it only told me when the lock has been opened.

The Lock'n'Go feature works as well, so I guess the door sensor itself is working. But the door sensor tile in the home app says closed all the time. 

Whenever I send a command from the home app, there is a little wheel spinning in the top right corner of the tile which does not stop so I guess there's something wrong with the request or its reply. Is there any way to debug and find where the information goes missing?

One more question: The name of the opener was pulled from the Nuki app or bridge. The name for the contact sensor tile and smart lock tile were shown as "Door". Why is it necessary to set a name for the door lock but not for the opener?

Best

Jan

```
{
      "platform": "NukiPlatform",
      "hostNameOrIpAddress": "spieglein",
      "hostCallbackApiPort": 40506,
      "bridgeIpAddress": "192.168.178.xxx",
      "bridgeApiPort": 8080,
      "bridgeApiToken": "xxx",
      "bridgeRebootSwitch": false,
      "devices": [
        {
          "nukiId": ...,
          "isDoorSensorEnabled": false,
          "isRingToOpenEnabled": false,
          "isContinuousModeEnabled": false,
          "isDoorbellEnabled": true,
          "isSingleAccessoryModeEnabled": false,
          "unlatchFromLockedToUnlocked": false,
          "unlatchFromUnlockedToUnlocked": false,
          "lockFromLockedToLocked": false,
          "unlatchLock": false,
          "unlatchLockPreventUnlatchIfLocked": false,
          "leaveOpen": false
        },
        {
          "nukiId": ...,
          "isDoorSensorEnabled": true,
          "isRingToOpenEnabled": false,
          "isContinuousModeEnabled": false,
          "isDoorbellEnabled": false,
          "isSingleAccessoryModeEnabled": false,
          "unlatchFromLockedToUnlocked": true,
          "unlatchFromUnlockedToUnlocked": true,
          "lockFromLockedToLocked": false,
          "unlatchLock": false,
          "unlatchLockPreventUnlatchIfLocked": false
        }
      ]
    }
```
(I removed some critical data, `xxx` and `...` is not actually there)

**Comment by lukasroegner** [comment]
Hi @nixmeer,

Whenever the plugin receives an update that the bell has been rang, you will see a log entry in the Homebridge logs: `Updating doorbell: Ring`

If that is not the case, you might have to check whether the callback (webhook) URL of the plugin has been registered properly. Just restart the plugin and take a look at the logs. If an error occurs while registering the callback URL on the Bridge, you should see a log entry.

Another thing you can check: perform an HTTP call to the callback list endpoint of the Nuki Bridge (`http://XXX.XXX.XXX.XXX:8080/callback/list?token=XXX`) to check whether your Homebridge host is properly registered.

**Comment by chrisleduc** [comment]
Hi @lukasroegner 

I'm running into the same issue. I'm using a bridge with version 1.17.0 (non-beta) and I don't get "Updating doorbell: Ring" in the log and accordingly also no reaction whatsoever.

Looking at the Nuki Bridge release notes, the feature is available to 1.17.0 but I can't seem to get it to work.

In homebridge I get:
[10/28/2020, 11:49:01 PM] [NukiPlatform] Callback server started.
[10/28/2020, 11:49:04 PM] [NukiPlatform] Callback already registered.

And on the bridge I see the host is registered. The only issue is that I'm not sure what port to use and left it as a default value.

Any thoughts?
